src/App.js

```diff
@@ -10,7 +10,6 @@ import { ToastContainer } from "react-toastify";
 import { SettingsContext } from "./settings";
 import AppRoutes from "./components/AppRoutes";
 import GlobalNav from "./components/GlobalNav/GlobalNav";
-import Footer from "./components/Footer/Footer";
```

Comment:
Is the Footer going to be added somewhere else?

Reply:
all the info and links that was in the footer is now in the sidebar info tab 😄